spec/features/user_signs_in_spec.rb

```diff
@@ -19,11 +19,53 @@
     expect(page).to have_content('Sign Out')
   end
 
-  scenario 'specify invalid credentials' do
+  scenario 'specify valid email' do
+    user = FactoryGirl.create(:user)
+
     visit new_user_session_path
 
+    fill_in 'Login', with: user.email
+    fill_in 'Password', with: user.password
+
     click_button 'Log in'
-    expect(page).to have_content('Invalid email or password')
+
+    expect(page).to have_content('Signed in successfully')
+    expect(page).to have_content('Sign Out')
+  end
+  scenario 'specify invalid credentials - empty fields' do
+    visit new_user_session_path
+
+    click_button 'Log in'
+    expect(page).to have_content('Invalid login or password')
     expect(page).to_not have_content('Sign Out')
   end
+
+  scenario 'specify invalid credentials - email/password combo not in database' do
+
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+
+    fill_in 'Login', with: user.email
+    fill_in 'Password', with: "Foobar1235"
+
+    click_on 'Log in'
+    expect(page).to have_content('Invalid login or password')
+
+  end
+
+  scenario 'specify invalid credentials - username/password combo not in database' do
+
+
+    user = FactoryGirl.create(:user)
+
+    visit new_user_session_path
+
+    fill_in 'Login', with: user.username
+    fill_in 'Password', with: "Foobar1235"
+
+    click_on 'Log in'
+    expect(page).to have_content('Invalid login or password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.